chia/wallet/wallet_node.py

```diff
@@ -379,11 +380,11 @@ async def _messages_to_resend(self) -> List[Tuple[Message, Set[bytes32]]]:
 
         return messages
 
-    async def _retry_failed_states(self):
+    async def _retry_failed_states(self) -> None:
         while not self._shut_down:
             try:
                 await asyncio.sleep(5)
-                current_time = time.time()
+                current_time = int(time.time())
```

Comment:
Note that we assign `self.last_state_retry_time = int(time.time())` at line
https://github.com/Chia-Network/chia-blockchain/pull/14390/files#diff-8387032f76cb8397134e314bda2f2ebbe6dd1c4aa473feac200bdbd4f27051b4L278

Reply:
I saw some inconsistency like this somewhere recently.  Happy to have it cleaned up.  But...  (of course `:]`) Do we actually want to drop the float resolution down to just integer seconds?

Reply:
I don't think we need to. But the reason I went in this direction was to not change the behaviour of a part of the code I'm not working on right now.

Reply:
Yep, didn't mean that should be considered to change here.  I'm just not sure why we dislike sub-second time resolution.